tex/gregoriotex-spaces.tex

```diff
@@ -1059,45 +1020,14 @@
 {\catcode`p=12 \catcode`t=12 \gdef\gre@makenum#1pt{#1}}%
```

Comment:
`\gre@makein` isn't needed anymore.

Might it make more sense to define `\strip@pt` in gregoriotex.tex (the way we do for a couple of other LaTeX primitives) and then use that instead of creating `\gre@makenum` which has an admittedly more complected usage syntax (`\expandafter\gre@makenum\the` vs. `\strip@pt`)?